### PR TITLE
Links

### PR DESCRIPTION
**Pull request by andrewrady**
#3

### CODE REVIEW

File: src/preload-hover.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ export default class PreloadHover {
   constructor(configuration = null) {
     const defaultConfiguration = {
       defaultDomScope: [document.body],
-      debounceTime: 50
+      debounceTime: 50,
+      linkType: true,
```
Comment (kendaleiv) [inline src/preload-hover.js]: Could you make `linkType` a string defaulting to `local`? Being a boolean I wouldn't know what `true`/`false` means.

It should accept `local` and `external` and throw if one of those expected values is not provided.

File: src/preload-hover.js [hunk 2]
```diff
@@ -25,14 +26,26 @@ export default class PreloadHover {
...
             }, this.configuration.debounceTime);
+          } else {
+            if(link.hostname != window.location.hostname) {
+              timer = setTimeout(() => {
+                const preload = document.createElement('link');
+
+                preload.setAttribute('rel', 'preload');
+                preload.setAttribute('href', link.href);
+
+                head.appendChild(preload);
```
Comment (kendaleiv) [inline src/preload-hover.js]: Could you move this into a method or similar to avoid code duplication?

Comment (andrewrady) [inline src/preload-hover.js]: Yeah I can change the boolean type and refactor the code. I didn't mean to merge the code so I am going to move it back as I work on those.

File: src/preload-hover.js (hunk 1 above)
Comment (andrewrady) [inline src/preload-hover.js]: Do we want to add a string for both?

Comment (kendaleiv) [inline src/preload-hover.js]: Yes, good call. :smile:

**Pull request merged**